python/paddle/fluid/framework.py

```diff
@@ -1479,6 +1479,8 @@ def clone(self, for_test=False):
             p = self.inference_optimize()
         else:
             p = Program()
+            p.current_block_idx = self.current_block_idx
+            p._seed = self._seed
```

Comment:
what is the meaning of ```_seed ``` and ```current_block_idx ```

Reply:
To keep the current_block_idx and seed of new program the same with the original program